app/lib/groups/src/pages/course/course_card.dart

```diff
@@ -17,39 +17,44 @@ import 'package:sharezone/groups/src/pages/course/course_details.dart';
 import 'package:sharezone/groups/src/pages/course/course_details/course_details_bloc.dart';
 import 'package:sharezone/groups/src/pages/course/course_edit/course_edit_page.dart';
 import 'package:sharezone/groups/src/widgets/group_share.dart';
-import 'package:sharezone_widgets/adaptive_dialog.dart';
-import 'package:sharezone_widgets/state_sheet.dart';
-import 'package:sharezone_widgets/theme.dart';
-import 'package:sharezone_widgets/widgets.dart';
+import 'package:sharezone_widgets/sharezone_widgets.dart';
 
 Future<bool> showCourseLeaveDialog(
     BuildContext context, bool isLastMember) async {
   return showLeftRightAdaptiveDialog<bool>(
-      context: context,
-      right: isLastMember
-          ? AdaptiveDialogAction.delete
-          : AdaptiveDialogAction(
-              title: "Verlassen",
-              isDefaultAction: true,
-              isDestructiveAction: true,
-              popResult: true,
-              textColor: Colors.red,
-            ),
-      defaultValue: false,
-      title: "Kurs verlassen${isLastMember ? " und löschen?" : "?"}",
-      content: Text(
-          "Möchtest du den Kurs wirklich verlassen? ${isLastMember ? "Da du der letzte Teilnehmer im Kurs bist, wird der Kurs gelöscht." : ""}"));
+    context: context,
+    right: isLastMember
+        ? AdaptiveDialogAction.delete
+        : AdaptiveDialogAction(
+            title: "Verlassen",
+            isDefaultAction: true,
+            isDestructiveAction: true,
+            popResult: true,
+            textColor: Colors.red,
+          ),
+    defaultValue: false,
+    title: "Kurs verlassen${isLastMember ? " und löschen?" : "?"}",
+    content: Text(
+      "Möchtest du den Kurs wirklich verlassen? "
+      "${isLastMember ? "Da du der letzte Teilnehmer im Kurs bist, wird der Kurs gelöscht." : ""}",
+    ),
+  );
 }
 
 Future<bool> showDeleteCourseDialog(
     BuildContext context, String courseName) async {
   return await showLeftRightAdaptiveDialog<bool>(
-      context: context,
-      right: AdaptiveDialogAction.delete,
-      defaultValue: false,
-      title: "Kurs löschen?",
-      content: Text(
-          'Möchtest du den Kurs "$courseName" wirklich endgültig löschen?\n\nEs werden alle Stunden & Termine aus dem Stundenplan, Hausaufgaben und Einträge aus dem Schwarzen Brett und gelöscht.\n\nAuf den Kurs kann von niemanden mehr zugegriffen werden!'));
+    context: context,
+    right: AdaptiveDialogAction.delete,
+    defaultValue: false,
+    title: "Kurs löschen?",
+    content: Text(
+      'Möchtest du den Kurs "$courseName" wirklich endgültig löschen?\n\n'
+      'Es werden alle Stunden & Termine aus dem Stundenplan, Hausaufgaben '
+      'und Einträge aus dem Schwarzen Brett und gelöscht.\n\n'
+      'Auf den Kurs kann von niemanden mehr zugegriffen werden!',
+    ),
+  );
```

Comment:
I guess this was done by some auto-formatting on your end? This change was made in several places in this PR.   

I mean it's not a bad change per se but as we don't use that kind of String formatting currently I would be more comfortable if you could revert these kind of changes in this PR.